docs/content/getting-started/production-best-practices.mdx

```diff
@@ -14,16 +14,25 @@ import {
 
 <DocumentationNotice />
 
-The following list outlines some guidelines and best practices for running OpenFGA in a production environment:
+The following list outlines best practices for running OpenFGA in a production environment:
 
 - [Configure Authentication](./setup-openfga/docker-setup.mdx#configuring-authentication)
 - Enable HTTP TLS or gRPC TLS or both
-- Set the log.format to "json"
-- Set the log.level to "info"
-- [Disable the Playground](./setup-openfga/docker-setup.mdx#playground).
+- Set the log format to "json" and log level to "info"
+- [Disable the Playground](./setup-openfga/docker-setup.mdx#playground)
+- [Set Cluster](#cluster-recommendations)
 - [Set Database Options](#database-recommendations)
 - [Set Concurrency Limits](#concurrency-limits)
 
+## Cluster Recommendations
+
+We recommend:
+
+1. Turn on caching via the flag `--check-query-cache-enabled`. This will reduce latency of requests, but it will increase the staleness of OpenFGA's responses.
```

Comment:
Do we want to mention that you can configure how low we cache for?

Reply:
I didn't want to put _too_ much information, my guess is that people can discover all the flags themselves.

Reply:
Ok, added a note